build.gradle

```diff
@@ -1586,15 +1605,17 @@ project(':streams:test-utils') {
   archivesBaseName = "kafka-streams-test-utils"
 
   dependencies {
-    compile project(':streams')
-    compile project(':clients')
+    implementation project(':streams')
```

Comment:
I guess this is similar to `connect-runtime` that it is easy to be used in users' testing scope. Maybe we should keep exposing both modules.

Reply:
Yes, agreed. This is even more important since this module is public API and it exposes classes from streams and clients in said API (e.g. `Deserializer` and `TopologyException`).